modules/react-components/src/components/input/file-upload.tsx

```diff
@@ -303,10 +307,13 @@ export const FileUpload: FunctionComponent<FileUploadPropsInterface> = (
                                     <p className="description">Drag and drop file here</p>
                                     <p className="description">– or –</p>
                                 </div>
-                                <Button basic primary onClick={ (event) => {
-                                    event.preventDefault();
-                                    fileUpload.current.click();
-                                } }>
+                                <Button
+                                    basic
+                                    primary
+                                    onClick={ (event) => {
+                                        event.preventDefault();
+                                        fileUpload.current.click();
+                                    } }>
```

Comment:
Shall we change as 
```
<Button
     onClick={ () => {                      
     } }
>
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/2712/commits/822ffa6c9b9d9f29aad197efcd574ef27c3e7440 & https://github.com/wso2/identity-apps/pull/2712/commits/f04b53ae80f3c688374854052c552a5f698db22f.